packages/mds-cache/redis/index.ts

```diff
@@ -63,6 +63,8 @@ export const RedisCache = () => {
 
     set: async (key: KeyType, val: ValueType) => safelyExec(theClient => theClient.set(key, val)),
 
+    mset: async (data: { [key: string]: ValueType } | string[]) => safelyExec(theClient => theClient.mset(data)),
```

Comment:
I wish TypeScript allowed a more expressive way to indicate that the array elements are an alternating sequence of `string` (key) and `ValueType` (value).

Reply:
yeah, that `| string[]` breaks my heart a little bit. Some kind of `nth-child` interface would be interesting.